src/main/java/dev/j3fftw/litexpansion/NanoBladeActiveEnchant.java

```diff
@@ -53,7 +56,14 @@ public boolean conflictsWith(Enchantment enchantment) {
     }
 
     @Override
-    public boolean canEnchantItem(ItemStack itemStack) {
-        return SlimefunUtils.isItemSimilar(itemStack, Items.NANO_BLADE, false);
+    public boolean canEnchantItem(ItemStack item) {
+        if (item.hasItemMeta()) {
+            ItemMeta itemMeta = item.getItemMeta();
+            Optional<String> id = SlimefunPlugin.getItemDataService().getItemData(itemMeta);
+            if (id.isPresent()) {
+                return (id.get().equals((Items.NANO_BLADE).getItemId()));
```

Comment:
Redundant brackets here

```suggestion
                return id.get().equals(Items.NANO_BLADE.getItemId());
```